src/vdaf/prio3.rs

```diff
@@ -130,6 +136,33 @@ impl Prio3Aes128Sum {
     }
 }
 
+/// The fixed point vector sum type. Each measurement is a vector of 16-bit fixed point decimals
+/// with 15 fractional digits and the aggregate is the sum. The verification function ensures the
+/// L2 norm of the vector is <= 1.
+#[cfg(feature = "fixed")]
+#[cfg_attr(docsrs, doc(cfg(feature = "fixed")))]
+pub type Prio3Aes128FixedPointL2BoundedVecSum =
+    Prio3<FixedPointL2BoundedVecSum<FixedI16<U15>, Field64>, PrgAes128, 16>;
```

Comment:
1. The size of the field needs careful consideration, as the soundness error depends on it. Given that the number of gadget calls depends on the dimension of the input vector, I'd say Field128 is a safer choice. Note that we may end up going with smaller fields in the future, but this depends on security analysis; see https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/102.
2. What's the likelihood that we're going to want more precision than 16 bits in the future? If this is likely, then consider reflecting the precision in the name somehow.

Reply:
1. What kind of soundness error do you refer to? We only checked that all number that might occur during validation fit into the field. Is there something else we need to be aware of when choosing the field size?
2. It is pretty likely, at least for 32 bits and possibly 64 bits. We will update the name.

Reply:
1. The "soundness error" refers to the probability that an invalid input gets deemed valid by the Aggregators. This is non-zero for any FLP system (i.e., any circuit one specifies for use in Prio3). The exact error depends on the circuit. Generally, it increases as the number size of the proof increases and decreases as the field size grows. The bound  can be roughly inferred from [BBCGGI19, Theorem 4.3], but no one has yet worked out the precise bound for Prio3. This is work in progress, DM if you're interested.
2. You might want to also make the type generic in the number representation (i.e., some implementation of `Fixed`).